packages/cosmos-tigergraph/src/index.ts

```diff
@@ -0,0 +1,185 @@
+import { InputNode, InputLink } from '@cosmograph/cosmos'
+
```

Comment:
@GenericP3rson There are a few ESLint errors in the code, can you address them please?

Reply:
Yep! Everything should be conforming to ESLint standards in commit https://github.com/cosmograph-org/cosmos-integrations/pull/1/commits/83eff4e6b9e951d6c1702e2faaba5c2720fbd480.